k8s/operators/src/pool/main.rs

```diff
@@ -664,6 +674,35 @@ async fn pool_controller(args: ArgMatches) -> anyhow::Result<()> {
             std::process::exit(1);
         }
     }
+    if let Ok(mut v1_alpha1_pools) = list_v1alpha1(&k8s, namespace, PAGINATION_LIMIT).await {
+        if !v1_alpha1_pools.is_empty() {
+            for dsp in v1_alpha1_pools.iter_mut() {
+                let res_ver = dsp.resource_version().unwrap();
+                let name = dsp.name_any();
+                let node = dsp.spec.node();
+                let disk = dsp.spec.disks();
+                if let Err(e) = create_v1beta1(
+                    &k8s,
+                    &name,
+                    namespace,
+                    Some(res_ver.clone()),
+                    NewDiskPoolSpec::new(node, disk),
+                )
+                .await
+                {
+                    error!("failed to create v1beta1 cr for v1alpha1 cr: {:?}", e);
+                } else {
+                    info!("cr creation successful :{}", name);
+                }
+            }
+        }
+    } else {
+        info!("ERROR: while listing for conversion");
+    }
+
+    info!("Sleeping for 5 secs");
+    sleep(Duration::from_secs(5));
```

Comment:
Do we need to sleep?

Reply:
Probably not required. Its just after we replace CRs with v1beta1 so thought of sleeping for 5s before removing v1alpha1. Ill try without sleep and check.

Reply:
Removed sleep

Reply:
Removed sleep